blaze/avatar.js

```diff
@@ -29,8 +29,14 @@ Template.avatar.helpers({
   },
 
   initialsCss: function () {
+    var user = this.user ? this.user : Meteor.users.findOne(this.userId);
     var css = '';
-    if (this.bgColor)  css += 'background-color: ' + this.bgColor + ';';
+
+    var backgroundColorProperty = 'background-color: ';
```

Comment:
Can to also add the equivalent code to the React template's `render` function? If not, I can add that afterwards.